Source/DiscoveryV1/Models/QueryResult.swift

```diff
@@ -42,11 +42,6 @@ public struct QueryResult: Codable, Equatable {
      */
     public var resultMetadata: QueryResultMetadata?
 
-    /**
-     Automatically extracted result title.
-     */
-    public var title: String?
-
```

Comment:
Why did `title` disappear from this model?  Technically this is a breaking change.